test/async-await.js

```diff
@@ -245,3 +246,49 @@ tap.test('async-error', function (t) {
         t.end();
     });
 });
+
+tap.test('async-bug', function (t) {
+    runProgram('async-await', 'async-bug.js', function (r) {
+        var stdout = r.stdout.toString('utf8');
+        var lines = stdout.split('\n');
+        lines = lines.filter(function (line) {
+            return ! /^(\s+)at(\s+)<anonymous>$/.test(line);
```

Comment:
can you elaborate on why these lines need filtering out?

Reply:
I guess they don't that's just copy paste from a different test block, i glossed over it.

Reply:
Ok, removed the copy/paste stuff.